rules/autoconsent/onlyFans.json

```diff
@@ -11,7 +11,8 @@
       "click": "div.b-cookies-informer__nav > button:nth-child(1)"
     },
     {
-      "click": "div.b-cookies-informer__switchers > div:nth-child(2) > div[at-attr=\"checkbox\"] > span.b-input-radio__container > input[type=\"checkbox\"]"
```

Comment:
Should we add a `urlPattern` on this rule? In the coverage data it only runs on onlyfans.com.